Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -357,34 +357,6 @@ void NativeReanimatedModule::unregisterEventHandler(
       [=] { eventHandlerRegistry->unregisterEventHandler(id); });
 }
 
-jsi::Value NativeReanimatedModule::getViewProp(
```

Comment:
Due to this PR - https://github.com/software-mansion/react-native-reanimated/pull/4929 we should restore this method.